include/nbl/video/IDriverMemoryBacked.h

```diff
@@ -27,6 +27,17 @@ class IDriverMemoryBacked : public virtual core::IReferenceCounted
             uint32_t prefersDedicatedAllocation     : 1; /// Used and valid only in Vulkan
             uint32_t requiresDedicatedAllocation    : 1; /// Used and valid only in Vulkan
         };
+        
+        struct SDriverMemoryRequirements2
+        {
+            size_t   size;
+            uint32_t memoryTypeBits;
+            uint32_t alignmentLog2 : 6;
+            uint32_t prefersDedicatedAllocation     : 1;
+            uint32_t requiresDedicatedAllocation    : 1;
+        };
+        static_assert(sizeof(SDriverMemoryRequirements2)==16);
```

Comment:
why have a struct with a `2` prefix?

Arent we nuking the old one?

Reply:
Yea please don't review right now :)))
I'm going to. I just didn't want the hassle to change every place to fix compile errors, after I get this working I'll remove the old one and rename